src/components/createElementComponent.tsx

```diff
@@ -114,14 +114,15 @@ const createElementComponent = (
       }
     }, [options]);
 
-    React.useEffect(
-      () => () => {
-        if (elementRef.current) {
-          elementRef.current.destroy();
+    React.useEffect(() => {
+      const element = elementRef.current;
```

Comment:
This change was [recommended](https://reactjs.org/blog/2020/08/10/react-v17-rc.html#potential-issues) but not actually necessary. Made it in case it prevents any future bugs.